osconfig_tests/test_suites/package_management/package_management.go

```diff
@@ -58,6 +59,19 @@ type packageManagementTestSetup struct {
 	vf         func(*compute.Instance, string, int64, time.Duration, time.Duration) error
 }
 
+func NewPackageManagementTestSetup(image, name, fname, vs string, oc *osconfigpb.OsConfig, assignment *osconfigpb.Assignment, startup *api.MetadataItems, vf func(*compute.Instance, string, int64, time.Duration, time.Duration) error) *packageManagementTestSetup {
+	return &packageManagementTestSetup{
+		image:      image,
+		name:       name,
+		osconfig:   oc,
+		assignment: assignment,
+		fname:      fname,
+		vf:         vf,
+		vstring: 		vs,
```

Comment:
minor: looks like the spacing is off here.

Reply:
done